server/src/routes/tasksRoute.ts

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+import getTasksSection from '../controllers/projects';
+
+const tasksRoute = express.Router();
+tasksRoute.post('/tasks/:id', getTasksSection);
+
```

Comment:
The method is GET not POST.
and I think the route is ```/section/:id``` because you need sectionId, not taskId
(we get tasks assigned in this section)

Reply:
yup , i fixed it and solve you can see the edit 
we write this route as we agreed and write it in hackmd but I will edit it